### ISSUE TITLE
Unique content on "Products" page depending upon what bucket you are in...

### ISSUE THREAD

**Issue by pvankirk** [realbig/TH_APPLEGATE#46]
Kyle,

Have you decided on how you wanted to approach this? We had discussed a couple of possibilities, but we will need to make a decision as it impacts other parts of the site navigation (e.g. the footer).

Thanks,
Paul

**Comment by brashrebel** [comment]
Currently the bucket association is being handled with session data. What this means is that as soon as a user identifies themselves as a certain persona by clicking or visiting any page inside one of the 4 persona buckets, that persona will be stored in their session. That value will be used for the automatic filtering of things like products and FAQs when they visit those pages. Does this sound acceptable?

**Comment by pvankirk** [comment]
While I agree this is a logical approach, there are cases where this doesn't work (e.g. media, sales, purchasing, etc.) where they may need unique information for a unique audience, but may not close out an existing session or be ignorant to where they should go first

 Another concern is how someone gets to our site. If they find us through a search because a less-relevant page from an incorrect bucket indexes better than the correct bucket we will pigeon-hole the user and they won't be able to reach the information that applies to them, even if they return to the home page to get to the right bucket.

In my conversations with Kyle, we discussed the merits of an elegant solution vs. driving nails with a 10 lb sledgehammer. If it requires an excessive amount of effort to develop an elegant solution, we can always fall back to making unique pages with unique content. This would be less than ideal, but could be our "best" solution.

**Comment by brashrebel** [comment]
I understand your concern with this. Though it is my belief that the actual use on this site of this feature is minimal and not so obtrusive that someone finding themselves accidentally in the wrong bucket would be unable to easily access the information they are seeking. The main places this bucket identifier are used is when someone visits an FAQ or a products listing page. When someone visits one of those pages there will be a simple list of the categories which can be used to filter the results and it will be clear that one is already selected. This means users will easily be able to expand the list or select another category. I hope this makes sense.

**Comment by pvankirk** [comment]
I think this needs a deeper discussion. The products shown as well as the content on those pages will vary depending upon what bucket you are in. I would be OK with switching ID based upon clicking through the bucket icon on the home page, but we cannot lock someone down for an session. I also don't want to force them to identify themselves a second time.Let me think on this and I will get back tomorrow.

**Comment by brashrebel** [comment]
Notes for clarification:
- Users are not locked down to a bucket in any way with the current configuration. If they enter another bucket, that becomes their current bucket.
- The only places where this bucket session value is used is on the products page and the FAQs page. 
- Those pages, when visited without a bucket defined for the current user, will simply list everything with options available for filtering the results.
- But when a visitor hits those pages and they _do_ have a bucket defined then the results will simply be filtered already for them.

**Comment by pvankirk** [comment]
OK, that should be good, but the products page is a list while the individual products will be on their own pages in the Products bucket and the content will be unique to those pages as well.

When we list information for a homeowner regarding a specific product, the benefit message is comfort, value, safety and peace of mind. Architects will hear a similar message with a stronger focus on sound attenuation, LEED points, Life Cycle Analysis and technical data. Contractors will get a message about ease to install, product differentiation, support and other characteristics intrinsically important to someone on the sales end of the transaction. All of the content on the pages in the products category need to be able to be assigned to a bucket. as the language, graphics and documents may vary to a great degree.

So, our plan of attack I think needs to be: remove the products bucket graphic and products link in the header. This will eliminate any concern about what content needs to appear when it is clicked and we take control back over the message. The challenge that will make for you is how do I assign the products page to the remaining 4 buckets so it shows in the footer navigation and in the bucket navigation when you enter one of the buckets.

Can we associate a page with more than one bucket?  If so, I would think the products pages would be OK since they would be associated with the Products page.

Please let me know your thoughts on this.

**Comment by brashrebel** [comment]
@pvankirk we may need to discuss this. Without lengthy reflection or testing, it seems to me that the most obvious solution would be to create additional products. So if the case is that a single product needs to exist in multiple buckets and show different text in either one, then it might just make sense to make different products. They are the same product to you but managed separately. You could then have "Product A - Contractor" or...something like that to distinguish it from "Product A - Homeowner" or some other naming convention. Just a thought but it seems like it would solve a LOT of potential technical issues.

**Comment by pvankirk** [comment]
I agree. As I mentioned to Kyle previously, I'd love to have the elegant solution, but if it takes more time and effort to do that knowing our product line doesn't change very often at all, it just doesn't make much sense.

So, to confirm, we are removing "Products" bucket link from the header and the graphic for products from the home page. I will create multiple "Products" category pages and multiple item pages as well.

**Comment by brashrebel** [comment]
Sounds great. We'll take care of removing those items if you can get moving on that content and let us know if everything works the way you want it to.
